.github/workflows/pr-update.yaml

```diff
@@ -12,15 +12,20 @@ jobs:
   autoupdate:
     permissions:
       contents: write
-      pull-requests: write
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Check secret
+        id: checksecret
+        uses: ./.github/actions/is-defined
+        with:
+          value: ${{ secrets.PR_UPDATE_TOKEN }}
       - name: Automatically update PR
+        if: steps.checksecret.outputs.result == 'true'
         uses: adRise/update-pr-branch@437fab6e0ac7d2a668f2c479f64225edd7f303fd # v0.6.0
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.PR_UPDATE_TOKEN }}
```

Comment:
What's broken?

Reply:
When using `GITHUB_TOKEN` the workflows are not triggered.

Reply:
GH does this to minimise the risks of an infinite loop...

Reply:
I think I saw the auto-merge earlier.

Reply:
For reference https://github.com/orgs/community/discussions/33804

Reply:
> When you use the repository's GITHUB_TOKEN to perform tasks, events triggered by the GITHUB_TOKEN, with the exception of workflow_dispatch and repository_dispatch, will not create a new workflow run. This prevents you from accidentally creating recursive workflow runs. For example, if a workflow run pushes code using the repository's GITHUB_TOKEN, a new workflow will not run even when the repository contains a workflow configured to run when push events

Reply:
ok yeah, I did sync the branch once manually.